### DISCUSSION TITLE
CSS Form formatting help

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2007] [Help]
I am using a workaround to submit option picker inputs to save to my SQL DB  however the new form formatting does not match the old formating. could someone please help me at the very least get the top two drop-down menu text labels on the left of the input field like the others below it.   I have tried multiple variations of CSS formatting and can not get the label to the side. 
I would greatly appreciate your help.
![image](https://user-images.githubusercontent.com/80058238/125971020-88c0848f-2cb6-4c55-bbc0-27c20dfb0ae7.png)

**Comment by joebudi** [comment]
hey @AndrewJRamay . Wrap your top two inputs in a 'form group' component, and select 'Left' under Labels

**Reply by KernOil** [reply]
Hi, @joebudi I love simple fixes thank you, sir!
